content/cpp/concepts/constructors/constructors.md

```diff
@@ -16,75 +16,121 @@ CatalogContent:
 
 A **constructor** is a specially defined method in a C++ class that is automatically called when an instance of that class is created. It is typically used for tasks such as initializing class attributes in a new object. Like with functions, a constructor can take arguments that can aid in the initialization. The arguments are passed to the constructor method when a new object is created.
 
-### Syntax and Examples
+## Syntax
 
-A constructor method is defined within a class by using the `ClassName` followed by parentheses.
+A constructor method is defined within a class by using the `ClassName` followed by parentheses. It does not have a return type.
+
+```cpp
+class  ClassName {
+  public:
+    ClassName() {
+      // Constructor code
+    }
+};
+```
+
+## Examples
+
+- Basic Constructor
```

Comment:
I would suggest to use `###` or H3 for this heading as this is not a simple pointer but a subsection.

```suggestion
### Basic Constructor
```